18_regexp_lab/01.py

```diff
@@ -1,12 +1,12 @@
-"""
-Write a program that reads data
-from property files.
-Each line in the file can either be:
-    An empty line
-    A comment line (Start with #)
-    A property line (of the form key = value)
-
-Write a program that takes a property file name and key
-as command line arguments and prints the requested value
-"""
-
+import sys
+import re
+file = sys.argv[1]
+user_input = raw_input("please enter your input:")
+
+
+with open(file,'r') as f:
+    for line in f:
+        if user_input in line:
+            print re.sub(r'(\w+)\s?=','', line)
+
+

```

Comment:
That's quite far from what the program should do. It should get the two parameters on the command line (and not ask the user), but more importantly if the user asks for 'name' this will return some value from
`    # firstname = Foo`
but will not return from
`name     =   Foo`